mitodl/open-discussions
# avoid new user posts in notification

#### Pre-Flight checklist


#### What are the relevant tickets?
closes https://github.com/mitodl/open-discussions/issues/3395

#### What's this PR do?
This pr only includes posts in weekly and daily digests if it has been 24 hours since the post authors first post

#### How should this be manually tested?
Create a new user. 
Create a post by the new user. 
Create a post by an old user.
Run
```
from notifications.tasks import *
send_daily_frontpage_digests() (or send_weekly_frontpage_digests depending on your user's setting)
send_unsent_email_notifications()
```
Verify that you don't see the new user's post

You may need to delete some EmailNotification objects if you run this multiple times
